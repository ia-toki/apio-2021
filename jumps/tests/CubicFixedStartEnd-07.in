209636825ecd2c38299498cd26890ef71dd128c2
200 200
198 197 196 192 191 190 186 185 184 180 179 178 174 173 172 168 167 166 162 161 160 156 155 154 150 149 148 144 143 142 138 137 136 132 131 130 126 125 124 120 119 118 114 113 112 108 107 106 102 101 100 96 95 94 90 89 88 84 83 82 78 77 76 72 71 70 66 65 64 60 59 58 54 53 52 48 47 46 42 41 40 36 35 34 30 29 28 24 23 22 18 17 16 12 11 10 6 5 4 1 2 3 7 8 9 13 14 15 19 20 21 25 26 27 31 32 33 37 38 39 43 44 45 49 50 51 55 56 57 61 62 63 67 68 69 73 74 75 79 80 81 85 86 87 91 92 93 97 98 99 103 104 105 109 110 111 115 116 117 121 122 123 127 128 129 133 134 135 139 140 141 145 146 147 151 152 153 157 158 159 163 164 165 169 170 171 175 176 177 181 182 183 187 188 189 193 194 195 199 200
99 99 199 199
111 111 198 198
178 178 180 180
0 0 154 154
7 7 64 64
70 70 163 163
161 161 177 177
124 124 166 166
196 196 199 199
118 118 156 156
173 173 182 182
58 58 104 104
137 137 153 153
76 76 176 176
116 116 151 151
28 28 52 52
126 126 187 187
114 114 127 127
6 6 143 143
181 181 187 187
54 54 62 62
187 187 189 189
27 27 131 131
161 161 179 179
166 166 168 168
71 71 154 154
105 105 180 180
101 101 179 179
59 59 99 99
125 125 127 127
77 77 104 104
57 57 146 146
67 67 185 185
77 77 108 108
132 132 161 161
18 18 179 179
96 96 143 143
163 163 183 183
144 144 179 179
15 15 47 47
114 114 161 161
143 143 153 153
176 176 185 185
137 137 142 142
54 54 82 82
198 198 199 199
105 105 146 146
163 163 195 195
105 105 106 106
55 55 154 154
146 146 194 194
87 87 156 156
68 68 135 135
52 52 82 82
78 78 152 152
135 135 141 141
160 160 173 173
180 180 196 196
44 44 169 169
195 195 198 198
171 171 179 179
6 6 100 100
29 29 112 112
179 179 183 183
97 97 188 188
52 52 191 191
20 20 158 158
188 188 194 194
40 40 76 76
160 160 185 185
68 68 186 186
47 47 191 191
58 58 137 137
108 108 189 189
31 31 62 62
23 23 135 135
58 58 71 71
56 56 117 117
140 140 190 190
193 193 199 199
61 61 168 168
118 118 119 119
180 180 197 197
7 7 182 182
15 15 49 49
86 86 189 189
70 70 178 178
112 112 128 128
123 123 186 186
141 141 183 183
43 43 70 70
134 134 186 186
142 142 186 186
158 158 160 160
117 117 165 165
164 164 171 171
144 144 163 163
12 12 25 25
41 41 88 88
4 4 130 130
166 166 181 181
36 36 107 107
140 140 177 177
53 53 180 180
149 149 167 167
23 23 41 41
192 192 196 196
75 75 164 164
118 118 132 132
160 160 184 184
83 83 100 100
101 101 170 170
121 121 150 150
66 66 175 175
181 181 191 191
151 151 180 180
194 194 195 195
154 154 174 174
26 26 45 45
113 113 146 146
119 119 154 154
140 140 154 154
188 188 198 198
116 116 153 153
40 40 159 159
186 186 191 191
98 98 152 152
35 35 67 67
92 92 97 97
88 88 163 163
89 89 131 131
93 93 109 109
76 76 127 127
194 194 195 195
112 112 114 114
146 146 184 184
173 173 183 183
87 87 124 124
91 91 149 149
56 56 128 128
24 24 85 85
54 54 132 132
138 138 172 172
100 100 156 156
127 127 175 175
41 41 116 116
10 10 37 37
49 49 159 159
65 65 97 97
64 64 136 136
158 158 197 197
21 21 71 71
74 74 125 125
189 189 195 195
176 176 182 182
26 26 192 192
191 191 194 194
121 121 189 189
39 39 124 124
88 88 181 181
23 23 163 163
182 182 199 199
105 105 173 173
16 16 41 41
75 75 91 91
52 52 179 179
3 3 86 86
172 172 197 197
109 109 183 183
54 54 63 63
150 150 174 174
125 125 157 157
163 163 184 184
158 158 166 166
175 175 180 180
177 177 199 199
55 55 173 173
41 41 49 49
80 80 115 115
146 146 191 191
61 61 176 176
147 147 171 171
21 21 167 167
33 33 89 89
173 173 183 183
20 20 103 103
123 123 189 189
92 92 131 131
119 119 125 125
75 75 187 187
121 121 157 157
150 150 185 185
197 197 198 198
16 16 175 175
65 65 127 127
57 57 115 115
98 98 186 186
165 165 170 170
49 49 179 179
0 0 148 148
