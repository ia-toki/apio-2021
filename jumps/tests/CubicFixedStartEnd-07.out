fc5005cd102fffc504bc1615010ebf88823d65f3
OK
67
58
2
-1
-1
24
11
29
3
26
7
-1
11
37
24
-1
41
9
-1
5
-1
2
-1
13
2
18
50
53
-1
2
-1
4
37
-1
20
-1
28
14
24
-1
32
7
7
4
-1
1
28
22
1
8
33
30
3
-1
22
4
10
11
10
2
6
-1
-1
3
59
31
-1
5
-1
18
37
27
-1
54
-1
-1
-1
-1
34
5
21
1
12
-1
-1
51
34
12
42
28
-1
35
30
2
32
5
13
-1
-1
-1
11
-1
25
23
13
-1
3
28
10
17
-1
47
20
29
8
20
1
14
-1
23
24
10
7
25
1
4
35
-1
-1
36
15
3
4
1
2
26
7
9
29
-1
-1
-1
23
38
33
-1
-1
7
-1
2
27
-1
-1
4
5
13
3
46
-1
48
-1
12
46
-1
-1
23
-1
18
50
-1
16
22
15
6
4
15
21
-1
-1
31
27
16
-1
-1
7
-1
44
17
5
43
25
24
1
-1
-1
-1
57
4
21
-1
